config/webhook/manifests.yaml

```diff
@@ -3,20 +3,21 @@ apiVersion: admissionregistration.k8s.io/v1
 kind: MutatingWebhookConfiguration
 metadata:
   creationTimestamp: null
-  name: mutating-webhook-configuration
+  name: webhook-admission
 webhooks:
   - admissionReviewVersions:
       - v1
+      - v1beta1
     clientConfig:
       service:
         name: webhook-service
         namespace: system
-        path: /mutate-scripts-tests-testkube-io-v1-script
```

Comment:
I beleive, we still have scripts crd, it's kind of legacy, might be we can remove it now